src/Symfony/Component/DependencyInjection/Tests/Dumper/XmlDumperTest.php

```diff
@@ -32,6 +33,8 @@
 
 class XmlDumperTest extends TestCase
```

Comment:
side question: do we want to keep the xml dumper?
it's used by some external tools AFAIK, so likely not

Reply:
FTR I gave a try on other formats: PHP dumper cannot dump a non-compiled container, and YAML dumper suffers from #60573

Reply:
See https://github.com/symfony/symfony/pull/60597 which might set us free from this concern.

Reply:
Yes, the XML dump of the container is used by every static analysis Symfony Plugin I know, including PHPStan, PhpStorm's Symfony plugin and Psalm. I don't think we can get rid of it (especially considering PhpStorm's plugin isn't written in PHP so probably can't parse the PHP serialized data nicely).

Reply:
See #61280 to remove the dependency on ext-xml for the XmlDumper

Reply:
Dumping console data in XML must be kept.